en/guide/routing.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: page
 title: Express routing
+description: Learn how to define and use routes in Express.js applications, including route methods, route paths, parameters, and using express.Router for modular routing.
```

Comment:
Learn how to define and use routes in Express.js applications, including route methods, route paths, parameters, and using Router for modular routing.